test/test_index.py

```diff
@@ -1218,6 +1218,31 @@ def test_index_add_non_normalized_path(self, rw_repo):
 
         rw_repo.index.add(non_normalized_path)
 
+    @with_rw_directory
+    def test_index_version_v3(self, tmp_dir):
+        tmp_dir = Path(tmp_dir)
+        with cwd(tmp_dir):
+            subprocess.run(["git", "init", "-q"], check=True)
```

Comment:
Would it be possible to use the `Git` type instead? That way one would benefit from all the 'extras', especially around finding the right Git binary.

Reply:
OK done